sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -269,7 +260,9 @@ def test_biqquery_read_fn_api_fail(self):
           'apache_beam.io.gcp.bigquery.ReadFromBigQuery.*'):
         with Pipeline(remote_runner,
                       PipelineOptions(self.default_properties)) as p:
-          _ = p | beam.io.Read(beam.io.BigQuerySource('some.table'))
+          _ = p | beam.io.Read(
+              beam.io.BigQuerySource(
```

Comment:
Is there any value in maintaining these tests that explicitly check whether a non-default behavior fails for Runner v2 ?

Reply:
discussing this in the next comment.